manifests/config.pp

```diff
@@ -67,9 +68,25 @@
         }
       }
       'systemd' : {
-        include 'systemd'
-        ::systemd::unit_file {'prometheus.service':
-          content => template('prometheus/prometheus.systemd.erb'),
+        $systemd_content = template('prometheus/prometheus.systemd.erb')
+        if $use_camptocamp_systemd {
+          include 'systemd'
+          ::systemd::unit_file {'prometheus.service':
+            content => $systemd_content,
+          }
+        } else {
+          file { '/etc/systemd/system/prometheus.service':
+            mode    => '0644',
+            owner   => 'root',
+            group   => 'root',
+            notify  => $systemd_notify,
```

Comment:
`$systemd_notify` is defined further up.  It was never taken out of the code in https://github.com/voxpupuli/puppet-prometheus/pull/90